mqtt_bridge/app.py

```diff
@@ -0,0 +1,100 @@
+import inject
+import paho.mqtt.client as mqtt
+import rclpy
+from rclpy.node import Node
+
+from .bridge import create_bridge
+from .mqtt_client import create_private_path_extractor
+from .util import lookup_object
+
+
+def create_config(mqtt_client, serializer, deserializer, mqtt_private_path):
+    if isinstance(serializer, str):
```

Comment:
`serializer` is of rclpy.Parameter type so this check is False when passing `json:..` in the yaml file, the following fixes the json serialization problem
```suggestion
    if isinstance(serializer.value, str):
```